content/en/service_management/case_management/view_and_manage/_index.md

```diff
@@ -7,6 +7,8 @@ further_reading:
   text: "Case settings"
 ---
 
+{{< img src="/service_management/case_management/view_and_manage/view_and_manage_overview.png" alt="Case Management page showing view of all cases, option to select status, and view assigned members" style="width:100%;" >}}
```

Comment:
I recommend having the image after the ## Overview section (but that's my personal preference).